R/dfm_core.R

```diff
@@ -127,26 +127,45 @@ dfm_core <- function(Y, m, p, FC, method, scale, logs, outlier_threshold, diffs,
   }
 
   # any reason why est$Kstore is in such a strange form? why not simple lists?
-  k_list <- lapply(seq(NROW(est$Kstore)), function(i) est$Kstore[i, 1, drop = FALSE][[1]])
+  # SL: These objects are arma::field<mat> in the Rcpp code, which works much better
+  # than a list internally. As I understand it, in R it is in fact already a list, just
+  # one in which all the objects are matrices.
+  
+  # k_list <- lapply(seq(NROW(est$Kstore)), function(i) est$Kstore[i, 1, drop = FALSE][[1]])
```

Comment:
Fair enough. It looks like a 1 col matrix with nested matrices. But ok for me, just haven't seen this before.